README.md

```diff
@@ -21,7 +23,51 @@ sbt antlr4:antlr4Generate # Generates source code for IntellIJ IDEA
 sbt clean compile test
```

Comment:
When I run this, I get these error messages:
```
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
[error] Database host name not set. Please see run instructions in README.md for providing a valid database hostname
```

Reply:
Ah, I should remove this line from `Getting Started`

Refer to the section titled `Unit Testing with MySQL in Docker`